_tools/check_doc_imports.ts

```diff
@@ -14,6 +14,7 @@ const EXCLUDED_PATHS = [
   ".git",
   ".github",
   "_tools",
+  "README.md",
```

Comment:
`deno task lint:doc-imports` passes in my env without this change. Is this really necessary?

Reply:
Strange, it failed before without this line. I just tested it again, and it's fine too. I've reverted it now.